### PR TITLE
Rename sleuthers to minions (Fixes #866)

### LINKED ISSUE #866: Rename sleuthers to minions
### Problem description
- Hard to spell
- Hard to say
- Is not a word

- [  ] Change in documentation
- [  ] Change in code

### PR DESCRIPTION
**Pull request by nahidakbar**
### What this PR does

Fixes #866

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)
-   [x] I've assigned this PR to someone (if you don't know who to assign it to, pick @AlexGilleran)

### CODE REVIEW

**Comment by nahidakbar** [comment]
Also: https://github.com/magda-io/magda-config/pull/1 && https://github.com/magda-io/magda-config-data.gov.au/pull/2

**Comment by AlexGilleran** [comment]
@nahidakbar could you put a db migration in to rename/retarget existing webhooks?

**Comment by nahidakbar** [comment]
@AlexGilleran added migrator script

File: magda-migrator-registry-db/sql/V1_9__Rename_sleth_to_minion.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    SET name = 'minion-format',
+        url = 'http://minion-format/hook'
+    WHERE name = 'sleuther-format';
+
+UPDATE webhooks
+    SET name = 'minion-linked-data-rating',
+        url = 'http://minion-linked-data-rating/hook'
+    WHERE name = 'sleuther-linked-data-rating';
+
+UPDATE webhooks
```
Comment (AlexGilleran) [inline magda-migrator-registry-db/sql/V1_9__Rename_sleth_to_minion.sql]: minion-framework is a framework used by minions, it doesn't have a webhook

**Comment by nahidakbar** [comment]
@AlexGilleran ping

**Comment by AlexGilleran** [comment]
@nahidakbar Looks like the build is failing

**Comment by nahidakbar** [comment]
@AlexGilleran

**Pull request merged**